agentex/templates/admin/agentex/allcalibrators.html

```diff
@@ -131,7 +131,7 @@
 		for(ds = 0 ; ds < this.decs.length ; ds++){
 		    
 			n = (this.meta.decisions[this.decs[ds]] ? this.meta.decisions[this.decs[ds]] : 0);
-			el.append('<div class="decision'+(n==0 ? " zero" : "")+'"><a href="{% url agentex.admin.allcalibrators_check planetid %}'+this.meta.sourceid+'/" class="'+this.decs[ds]+'" data="'+n+'">'+this.decisionImage(this.decisionFull(this.decs[ds]))+'<\/a>'+n+'<\/div>');
+			el.append('<div class="decision'+(n==0 ? " zero" : "")+'"><a href="{% url 'agentex_all_calib planetid' %}'+this.meta.sourceid+'/" class="'+this.decs[ds]+'" data="'+n+'">'+this.decisionImage(this.decisionFull(this.decs[ds]))+'<\/a>'+n+'<\/div>');
 			// Attach click events to the decision images
```

Comment:
That is not the correct way of forming a url. See [this section of the docs](https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#url)
The quotes should only be around the URL part and not the arguments.